pkg/app/server_test.go

```diff
@@ -207,6 +207,7 @@ func TestFullScrapeCycle(t *testing.T) {
 # HELP kube_pod_init_container_status_waiting Describes whether the init container is currently in waiting state.
 # HELP kube_pod_init_container_status_waiting_reason Describes the reason the init container is currently in waiting state.
 # HELP kube_pod_labels Kubernetes labels converted to Prometheus labels.
+# HELP kube_pod_nodeselector Describes the Pod nodeSelectors.
```

Comment:
should the metric name be kube_pod_nodeselector**s**?

Reply:
Seems reasonable to me

Reply:
updated the metric name as suggested :)